docs/tutorials/e2e/prerequisites/prerequisites.md

```diff
@@ -137,7 +137,8 @@ export NO_PROXY="localhost,127.0.0.1,::1"
 
 #### http (80)
 
-The port http (80) should not be used, but in case it will, you can apply the above hints just by replacing https by http.
+The port http (80) should not be used, but it will, you can apply the above hints just by replacing https by http.
+
```

Comment:
The markdown linter don't like multiple empty lines ;-) 
```suggestion
```

Reply:
Already fixed